MetaRock.rm

```diff
@@ -0,0 +1,148 @@
+Darwinia Grant Template
+
+## MetaRock.app
+
+### Project Description
+
+We are racing towards a future with an ever increasing existence in the digital realm. 
+Currently digital assets are largely limited to the trading of images, digitally native items such as NFT art. 
+The majority of those participating in this trading arena are limited to the tech world elite. We see this is a 
+very narrow use for this new technological tool and see a much wider spectrum of applications and users for digital assets. 
+
+MetaRock will offer a platform for everyday users and businesses to fully take advantage of the potential of these emerging technologies 
+in ways that are relevant to them. We are working proactively to solve a future problem, a gap in the market which will emerge as the use of 
+digital assets becomes more widespread. We are going to build.... a series of interfaces that are specifically designed for each asset class
+with a fluid logic and intuitive UI experience that will facilitate seamless adoption... accessible to anyone. 
+
+
+Focusing on 3D images
+We Aim to specialise in building a minting platform that business can digitise their real world assets making them metaverse compatible.
+MetaRock believes in the future all assets will be represented in the virtual world as well as in the physical world and will bridge the gap
+between the metaverse and the everyday individual.
+
+
+
+### Why is this project is good for the Darwinia ecosystem:
+
+MetaRock will build a market place on Darwinian and Crab network allowing Users to mint and sell NFTs natively in the Darwinia Ecosystem.
+Metarock will also allow Darwinia Network's existing community to start trading and selling NFT art and resources on the Darwinia and Crab Blockchains
+bringing utility to Ring and Crab.
+
+
+### How you will integrate this project into Darwinia:
+in the early stages Metarock will use the rmrk1.0 shortly followed by rmrk2.0 and ERC 721. Metarock is already minting NTFs on the Crab and Pangolin Blockchain 
+with Plans to build a community backed Liquidity pool to help build the communities local Dex liquidity. Once our Beta is released we will enable the community 
+and Darwinia to start minting and selling NFTs 
+
+
+### Team
+
+- Josiah Kotzur ( Darwinia Ambassador / Darwinia Development team )
+- James ( Software engineer )
+- Ela ( UI/UX Designer )
+- Pili ( Graphic Designer )
+
+
+
+### Legal Structure
+Currently Metarock has no defined legal structure other then a Dapp created by  Kotzur Marine Industries  Pty Ltd  trading as Blockchain Integration  ABN 41 126 551 379 
+
+
+
+### Team Code Repos
+
+- https://github.com/Doinglifewell (Josiah Kotzur)
+- https://github.com/TopCodeBeast/MetaRock_RMRK.git (James)
```

Comment:
I would like to confirm that MetaRock will be open sourced in MIT license?

Reply:
Metarock and its Beta release will be open source code with MIT license. However Metarock reserves the right to make some of its code private once Investors are onboard and Metarock 1.0 is released. legacy code will be released at a rate that will allow Metarock to maintain a competitive advantage amongst new competition.